content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -168,13 +168,14 @@ Here is an example set of resources users may want to put under object count quo
 The same syntax can be used for custom resources.
 For example, to create a quota on a `widgets` custom resource in the `example.com` API group, use `count/widgets.example.com`.
 
-When using `count/*` resource quota, an object is charged against the quota if it exists in server storage.
+When using such a resource quota (nearly for all object kinds), an object is charged
+against the quota if the control plane has a stored copy.
```

Comment:
"if the control plane has a stored copy." is not very clear what this means, may be something like: "if the object kind exists (is defined) in the control plane"